modules/xmpp/SDPUtil.js

```diff
@@ -443,6 +443,30 @@ var SDPUtil = {
             .split(" ")
             .map(ssrcStr => parseInt(ssrcStr));
     },
+
+    /**
+     * Sets the given codecName as the preferred codec by
+     *  moving it to the beginning of the payload types
+     *  list (modifies the given mline in place)
+     * @param {object} videoMLine the video mline object from
+     *  an sdp as parsed by transform.parse
+     * @param {string} the name of the preferred codec
+     */
+    preferVideoCodec: function(videoMLine, codecName) {
+        let payloadType = null;
```

Comment:
Chrome already has an open issue about adding H264 high profile together with baseline, which seems mostly ready for landing: https://bugs.chromium.org/p/webrtc/issues/detail?id=6337